src/Layer/TiledGeometryLayer.js

```diff
@@ -264,6 +275,20 @@ class TiledGeometryLayer extends GeometryLayer {
         return ObjectRemovalHelper.removeChildren(this, node);
     }
 
+    #hideExistingSkirt(value) {
+        for (const node of this.level0Nodes) {
+            node.traverse((obj) => {
+                if (obj.isTileMesh) {
+                    if (value) {
+                        obj.geometry.setDrawRange(0, obj.geometry.segments * obj.geometry.segments * 2 * 3);  //  bufferIndex = (nSeg) * (nSeg) * 2 * 3 (computeBufferTileGeometry.js)
+                    } else {
+                        obj.geometry.setDrawRange(0, Infinity);
+                    }
```

Comment:
See previous comment in `TileBuilder.js`, this could be a setter in `TileGeometry`.